src/helpers/clipboard/clipboard_figure_state.ts

```diff
@@ -84,12 +84,20 @@ export class ClipboardFigureState implements ClipboardState {
    */
   paste(target: Zone[]) {
     const sheetId = this.getters.getActiveSheetId();
+    const size = { height: this.copiedFigure.height, width: this.copiedFigure.width };
+    const viewportRect = this.getters.getMainViewportRect();
+    const colDimensions = this.getters.getColDimensions(sheetId, target[0].left);
```

Comment:
if you only plan on using one key of an object you can destructure it directly.
```suggestion
    const {start: colStart} = this.getters.getColDimensions(sheetId, target[0].left);
```
will do create the variable `colStart` for instance. It will also make the rest of the code more readable